core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -791,6 +795,25 @@ object IO extends Serializable {
     override def tag = Tags.Ensuring
   }
 
+  sealed abstract class Race private[zio] { self =>
```

Comment:
We should move the `private[zio]` to the front, so the whole class is private, otherwise just the constructor will be private.